src/main/webapp/css/styleguide.txt

```diff
@@ -0,0 +1,34 @@
+// Website Wide
+//  -->Tags
+//  --> h1 - 30px
+//  --> h2 - 
+//  --> h3 -
+//  --> p - 
```

Comment:
BTW a better way to do this would be to write this in a Markdown file. E.g. you could add it to the README.md and style it fancily like the things there.